### ISSUE TITLE
Importing/Exporting itineraries

### ISSUE THREAD

**Issue by KennethOyolla** [visitbasis/visitbasis-issues#21]
The importing and exporting itineraries, is it possible to include a divisional filter?
This would help say when exporting itineraries for a week for Golden Marketing, it exports itineraries for all divisions whilst in fact one most of the time wants to edit itineraries for one division only. I acknowledge that the tasksIDs are unique to a Division but then it requires the person importing/exporting to be able to differentiate the Ids ignorer to know what division the Ids relate to.

In summary the divisional ID  is a higher hierarchy that makes it easy to zero down on the tasks/visits to change

**Comment by visitbasis** [comment]
Hi Kenneth,

Thanks for the idea. I see how this might help. But this also will make the data denormalized and will be inconsistent with the design principles - not to duplicate any information when importing data. 

So we have to reject the for now.

**Comment by KennethOyolla** [comment]
Thanks. I understand 

Kind regards

Kenneth
